site/public/js/server.js

```diff
@@ -1666,7 +1666,13 @@ function addMarkdownCode(type){
     let insert = '';
     switch (type) {
         case 'code':
-            insert = '```\ncode\n```';
+            const current = text[cursor];
+            const last_newline_pos = text.substring(0, cursor).split('').lastIndexOf('\n');
+            if (text.substring(last_newline_pos, cursor).length !== 1) {
+                insert = '\n';
+            }
+            console.log('current', current);
```

Comment:
```suggestion
```

Was this being logged deliberately?

Reply:
No, I added that just to test, my most recent commit should have gotten rid of that

Reply:
Just fiddling with getting e2e tests to pass now

Reply:
Yeah, my bad.  GitHub didn't refresh or something and I didn't see the removal of the console log until after I submitted it.  Changes look good and seem to work as expected.